examples/hitl/pick_throw_vr/pick_throw_vr.py

```diff
@@ -59,6 +59,8 @@ def __init__(self, app_service: AppService):
         self._can_grasp_place_threshold = (
             self._app_service.hitl_config.can_grasp_place_threshold
         )
+        # Activate the remote user.
+        app_service.users.add_user(0)
```

Comment:
Again, it seems like you want to call this function `activate_user` based on this comment.

Also is there still a notion of a non-network mode for this PickThrowVR app? Does this line break something in that case?

Reply:
>Also is there still a notion of a non-network mode for this PickThrowVR app? Does this line break something in that case?

The `pick_throw_vr` application works just as before in local mode.